assets/src/edit-story/components/panels/design/preset/presetGroup.js

```diff
@@ -99,10 +100,11 @@ function PresetGroup({ presets, itemRenderer, type, handleClick, isEditMode }) {
     }
   }, [activeIndex, presets.length]);
 
+  const keys = useMemo(() => presets.map(() => uuidv4()), [presets]);
```

Comment:
Hm, this isn't right actually. The same item should have the same key always. With this, all the keys will be recalculated if any item changes (or is added/removed). We could perhaps just `JSON.stringify(preset)` or something like that?

Reply:
Hmm, we could, it could theoretically be a very long key then but I guess that's fine.

Reply:
We should apply a unique id to these presets at the story data level imo. I think this is fine for now tho and shouldn't lead to bugs even tho it should cause rerenders on every item everytime `presets` updates. that being said, I don't think we'll ever have so many presets on a story that this perf improvement would ever be impactful.

`JSON.stringify(preset)` could lead to duplicate keys if a user has added multiple of the same preset anywhere. I think we'd need `${JSON.stringify(preset)}-${i}` which would get us back to the eslint error.

It's kinda impossible to retroactively uniquely identify something down stream that's not uniquely identifiable

Reply:
> `JSON.stringify(preset)` could lead to duplicate keys if a user has added multiple of the same preset anywhere.

Presets should be unique though, there's currently a bug where in one case it's possible to add multiple of the same preset but that will get fixed in #5216